docs/api.md

```diff
@@ -4442,13 +4442,19 @@ This method returns boxes of the element, or `null` if the element is not visibl
   - `button` <"left"|"right"|"middle"> Defaults to `left`.
   - `clickCount` <[number]> defaults to 1. See [UIEvent.detail].
   - `delay` <[number]> Time to wait between `mousedown` and `mouseup` in milliseconds. Defaults to 0.
+  - `offset` <[Object]> Offset in pixels relative to the border box of the element.
+    - `x` <number> x-offset in pixels relative to the border box of the element.
+    - `y` <number> y-offset in pixels relative to the border box of the element.
```

Comment:
Relative to which corner (top left or bottom left)? Please clarify this in the docs.